app/views/profile/show.html.erb

```diff
@@ -11,5 +11,5 @@
   <p class="govuk-body">Roles: <%= @user.roles %></p>
   <p class="govuk-body">E-mail address: <%= @user.email %></p>
   <p class="govuk-body">Phone number: <%= @user.phone_number %></p>
-  <a href="/users/sign_out" data-method="delete" class="govuk-button" data-module="govuk-button">Logout</a>
+  <a href="/profile/edit" class="govuk-button" data-module="govuk-button">Edit Profile</a>
```

Comment:
Submit button could be more railsy: `<%= link_to "Edit Profile", profile_edit_path, class: "govuk-button" %>`